<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="head_label_1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>POP3 Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="pop_host">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>POP3 Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pop_host_value">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pop_port">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>POP3 Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_text">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="smtp_host">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SMTP Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="smtp_host_value">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="head_label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SMTP Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_status">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="smtp_port">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SMTP Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_text">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>71</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="app_title">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>PyMail</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="smtp_port_value">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pop_port_value">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pass_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>92</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>330</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>14114013</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>330</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Anshul Shah</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>390</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>14114028</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>390</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Harkirat Singh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>430</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>14114030</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>430</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ketan Mittal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nikhil Sheoran</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>14114031</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>410</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Prashant Kumar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>410</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>14114037</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Pratyush Rajput</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>370</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>14114039</string>
    </property>
   </widget>
   <widget class="QLabel" name="head_label_3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Created By</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1263</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <action name="actionSMTP_Host">
   <property name="text">
    <string>SMTP</string>
   </property>
  </action>
  <action name="actionSMTP_Port">
   <property name="text">
    <string>SMTP Port</string>
   </property>
  </action>
  <action name="actionPOP3_Host">
   <property name="text">
    <string>POP3</string>
   </property>
  </action>
  <action name="actionPOP3_Port">
   <property name="text">
    <string>POP3 Port</string>
   </property>
  </action>
  <action name="actionRetrieve_Mails">
   <property name="text">
    <string>Retrieve Mails</string>
   </property>
  </action>
  <action name="actionCompose">
   <property name="text">
    <string>Compose</string>
   </property>
  </action>
  <action name="actionCreated_By">
   <property name="text">
    <string>Created By</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
